Symfony/CS/Fixer/Symfony/NoEmptyLinesAfterPhpdocsFixer.php

```diff
@@ -26,13 +26,24 @@ class NoEmptyLinesAfterPhpdocsFixer extends AbstractFixer
      */
     public function fix(\SplFileInfo $file, $content)
     {
+        static $forbiddenSuccessors = array(
+            T_DOC_COMMENT,
+            T_COMMENT,
+            T_WHITESPACE,
```

Comment:
I don't understand why whitespace would be a reason to skip. If someone has left whitespace on an empty line, we don't want to screw up. We should fix it anyway.